lib/livebook_web/live/session_live/embedded_live.ex

```diff
@@ -0,0 +1,43 @@
+defmodule LivebookWeb.SessionLive.EmbeddedLive do
+  use LivebookWeb, :live_view
+
+  alias Livebook.{Session, Runtime}
+
+  @impl true
+  def mount(_params, %{"session_id" => session_id}, socket) do
+    {:ok, assign(socket, session_id: session_id)}
+  end
+
+  @impl true
+  def render(assigns) do
+    ~L"""
+    <div class="flex-col space-y-5">
+      <p class="text-gray-700">
+        Connect to a new runtime within the Livebook node itself.
+        This is reserved for specific cases where there is no option
+        of starting a separate Elixir runtime
+        (e.g. when running Livebook directly on a Nerves device).
```

Comment:
A tip: I was told by an editor that we should avoid e.g. and i.e., as they may not be very clear to non native speakers. :)

Reply:
Good to know!